views/new_peep.erb

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <h1>Hello, <%= @username %></h1>
+  </head>
+
+  <body>
+    <form action="/peep" method="POST">
+      <div>
+        <label>Create a peep</label><br />
+        <input maxlength="150" type="text" name="content"><br />
```

Comment:
Instead of text type, you could have used the textarea, so that user can clearly see their peeps. It's great you restrict the maximum length of the peep though.  I didn't know you could do that.